sdk/tests/TransactionRecordQueryTest.cc

```diff
@@ -58,4 +65,38 @@ TEST_F(TransactionRecordQueryTest, SetTransactionId)
   const TransactionId transactionId = TransactionId::generate(getTestAccountId());
   query.setTransactionId(transactionId);
   EXPECT_EQ(query.getTransactionId(), transactionId);
+}
+
+// Tests invoking of method execute() from TransactionRecordQuery.
+TEST_F(TransactionRecordQueryTest, ExecuteRequestToTestnetNode)
+{
+  // Given
+  const auto testMemo = "Test memo for TransactionRecordQuery.";
+  const auto testPublicKey = ED25519PrivateKey::generatePrivateKey()->getPublicKey();
+  const auto accountOperatorId = AccountId::fromString("0.0.1359");
+  Client testnetClient = Client::forTestnet();
+  testnetClient.setOperator(
+    accountOperatorId,
+    ED25519PrivateKey::fromString("2e000363977258a41f418cf84a7df9cf1e8ae98b72de86803e64846defa43054"));
+
+  AccountCreateTransaction testAccountCreateTransaction;
+  testAccountCreateTransaction.setKey(testPublicKey);
+  testAccountCreateTransaction.setTransactionMemo(testMemo);
+
+  TransactionId testTransactionId =
+    testAccountCreateTransaction.execute(testnetClient).getRecord(testnetClient).getTransactionId().value();
+
+  TransactionRecordQuery testTransactionRecordQuery;
+  testTransactionRecordQuery.setTransactionId(testTransactionId);
+  testTransactionRecordQuery.setMaxAttempts(3);
```

Comment:
What is the purpose of setting this here?

Reply:
You're correct - calling of `setMaxAttempts(3)` could be removed here too.

Reply:
Fixed in 585305dab4b71794413da99058d22d301f00fd26.